camunda-formio-tasklist-vue/src/components/layout/Header.vue

```diff
@@ -14,6 +14,7 @@
               :key="filter.id"
               href="#"
               @click="togglefilter(filter, idx)"
+              :active="idx == activefilter"
```

Comment:
I think the operator here should be `====`. FYR- https://howtodoinjava.com/typescript/equals-vs-strict-equals/